egs/babel/s5d/local/run_asr_segmentation.sh

```diff
@@ -0,0 +1,336 @@
+#! /bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+# Features configs
+feat_type=plp
+feat_config=conf/plp.conf
+add_pitch=true
+pitch_config=conf/pitch.conf
+
+# Uniform segmentation options
+max_segment_duration=10
+overlap_duration=2.5
+max_remaining_duration=5
+
+# Decoding options
+lang=data/lang
+lang_test=data/lang
+
+data_dir=data/train
+sat_model_dir=/export/b17/jtrmal/babel/104-pashto-flp80-p-ext/exp/tri5_cleaned
```

Comment:
I don't think we'd normally want to check things in with absolute filenames.     I'd be willing to make an exception, but at least there should be a comment saying you should replace it with something that exists on your system.  And I don't know how important it is which system it is?